tests/SpellChecker/SpellChecker.phpt

```diff
@@ -0,0 +1,49 @@
+<?php declare(strict_types = 1);
+
+namespace SpellChecker;
+
+use SpellChecker\Dictionary\DictionaryCollection;
+use SpellChecker\Dictionary\DictionaryResolver;
+use SpellChecker\Heuristic\DictionarySearch;
+use SpellChecker\Parser\DefaultParser;
+use SpellChecker\Parser\PhpParser;
+use Tester\Assert;
+
+require '../bootstrap.php';
+
+$phpParser = new PhpParser(new DefaultParser());
+$directory = '../../vendor/spell-checker/dictionary-en';
```

Comment:
`__DIR__`

Reply:
nejde - předává se tam před to `getcwd()`